docs/plugin.md

```diff
@@ -218,3 +219,7 @@ Site objects include a site diagram from IPFabric, which is using the new custom
 | vlanId             | Vlan.vid       | VLAN.vid               |
 | status             | Vlan.status    | VLAN.status            |
 | siteName           | Vlan.site      | VLAN.site              |
+
+## Device Roles
+
+Although IP Fabric does not have a dedicated model that corresponds to Nautobot Device Role objects, the plugin will create new Device Roles that correspond to the synced Device's `devType` attribute. The resulting Device Role object in Nautobot will have a custom field called **IPFabric Type** that maps directly to the `devType` field in IP Fabric. You can change the Device Role's name and color without any impact between syncs. However, if you change the custom field **IPFabric Type**, then on subsequent syncs a new Device Role object will be created and the Devices that have been synced from IP Fabric will revert to using the new Device Role.
```

Comment:
I think this description would benefit from a table like the other models. It's not immediately obvious to me when reading the text what attributes **cannot** be modified in order to maintain the existing association.
Also it would be useful to reference `name` and `color` as code to identify them as specific attributes.